http/vibe/http/server.d

```diff
@@ -657,6 +658,9 @@ final class HTTPServerSettings {
 	/// the url and all headers.
 	ulong maxRequestHeaderSize = 8192;
 
+        /// Maximum number of bytes in a single line in the request header.
+        ulong maxRequestHeaderLineSize = 4096;
```

Comment:
This should be `size_t` in order to be able to be passed to `parseFormData` on 32-bit builds. `maxRequestHeaderSize` should probably also be a `size_t`, since it needs to fully fit into memory, too, but that's a different issue.

Minor note: the indentation here uses spaces instead of tabs.